springfield/base/templates/404.html

```diff
@@ -6,6 +6,8 @@
 
 {% extends "base-error.html" %}
 
+{% set referrals = '?utm_source=www.firefox.com-404&utm_medium=referral' %}
```

Comment:
```suggestion
{% set referrals = '?utm_source=www.firefox.com&utm_medium=referral&utm_content=404' %}
```

Here's an example of what I meant when I commented we should standardize the source and rely on content for the pages.

Reply:
> I think we should standardize the utm_source to always be www.firefox.com and rely on the utm_campaign to communicate the page.

To confirm: should we use `utm_campaign` for page names (i.e. 404) and `utm_content` for partials (i.e. [newsletter-form](https://github.com/mozmeao/springfield/pull/290/files#diff-57dfde908cc6340e52c95b7de428a4b66628e8fc99413fe4003c7e4798e8a38b) or footer/top-nav)?